packages/dev/core/src/XR/features/WebXRBackgroundRemover.ts

```diff
@@ -60,7 +60,7 @@ export class WebXRBackgroundRemover extends WebXRAbstractFeature {
     constructor(
         _xrSessionManager: WebXRSessionManager,
         /**
-         * read-only options to be used in this module
+         * [Object] read-only options to be used in this module

```

Comment:
Is it not "Empty object"? Same comment x2 below.

Reply:
In fact, I meant to use "[Empty object]" instead of "[Object]": is it not allowed?

Reply:
Everything is allowed, it's just a string (in our case, not in JSDoc). I don't mind changing it